### ISSUE TITLE
is this project still active?

### ISSUE THREAD

**Issue by xmnlab** [bastikr/boolean.py#120]
hey everyone! 

I ran into this library, and it looks really nice.

it seems that the latest release was almost 2 years ago and I was wondering if this project is still active or there is any plan to keep this project moving forward.

all the best,
Ivan

**Comment by tomas789** [comment]
Hello Ivan! Sometimes the job is basically done. It won't stop working just because there is no recent change. Feel free to use it. 😉

**Comment by pombredanne** [comment]
> Hello Ivan! Sometimes the job is basically done. It won't stop working just because there is no recent change. Feel free to use it. 😉

Exactly. The project is actively used and is maintained with about 4 million downloads per month.
 Closing this now
